Formula/grv.rb

```diff
@@ -0,0 +1,34 @@
+class Grv < Formula
+  desc "Terminal interface for viewing git repositories"
+  homepage "https://github.com/rgburke/grv"
+  url "https://github.com/rgburke/grv/releases/download/v0.1.2/grv-0.1.2-src.tar.gz"
+  sha256 "1f2f8adb28085fd892f4b23cf5cf17925fc502cf479be35193e1c6b93cad1e49"
+  head "https://github.com/rgburke/grv.git"
+
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+  depends_on "pkg-config" => :build
+  depends_on "readline"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    path = buildpath/"src/github.com/rgburke/grv"
+    path.install buildpath.children
+
+    cd path do
+      system "make", "build-only"
+      bin.install "grv"
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.